### PR TITLE
Issue: 3375239 - add email scan in packages

### PR DESCRIPTION
**Pull request by NirWolfer**
Issue: [HPCINFRA-920](https://jirasw.nvidia.com/browse/HPCINFRA-920)

## Description
Add private email scan in deb/rpm packages

##### What
Make sure we dont have any private emails in rom/deb packages information or changelog

##### Why ?
RM3375239

## Change type
What kind of change does this PR introduce?
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [X] CI related changes
- [ ] Documentation content changes
- [ ] Tests
- [ ] Other

## Check list
- [ ] Code follows the style de facto guidelines of this project
- [ ] Comments have been inserted in hard to understand places
- [ ] Documentation has been updated (if necessary)
- [ ] Test has been added (if possible)

### CODE REVIEW

**Comment by NirWolfer** [comment]
bot:retest

**Comment by NirWolfer** [comment]
bot:restest

**Comment by NirWolfer** [comment]
bot:retest

**Comment by NirWolfer** [comment]
bot:retest

**Comment by NirWolfer** [comment]
bot:retest

**Comment by NirWolfer** [comment]
bot:retest

File: contrib/jenkins_tests/rpm.sh [hunk 1]
```diff
@@ -87,5 +87,38 @@ if [ $opt_checkpkg -eq 1 ]; then
...
+    rc=$((rc + 1))
+elif [[ -n "$test_output" ]]; then
+    # if we got rc not 0 and we have output it means something else failed
+    echo "ERROR: could not find bad email but something else failed: $test_output"
+    rc=$((rc + 1))
+fi
+
 echo "[${0##*/}]..................exit code = $rc"
-exit $rc
+exit $rc
```
Comment (dpressle) [inline contrib/jenkins_tests/rpm.sh:124]: Missing newline

**Review by dpressle — CHANGES_REQUESTED** [review]
missing

**Comment by dpressle** [comment]
please check your commits it seems like you have test commit that should be removed or squashed

File: contrib/jenkins_tests/rpm.sh (hunk 1 above)
Comment (NirWolfer) [reply contrib/jenkins_tests/rpm.sh]: Newline added

**Comment by NirWolfer** [comment]
Redmine test commit was squashed into the main commit

**Review by dpressle — APPROVED** [review]

**Pull request merged**